astrowidgets/tests/test_image_wdiget.py

```diff
@@ -25,3 +29,35 @@ def test_add_marker_does_not_modify_input_table():
     in_table = Table(data=[x, y], names=['x', 'y'])
     image.add_markers(in_table, pixel_coords_offset=5)
     assert (in_table == orig_table).all()
+
+
+def test_adding_markers_as_world_recovers_with_get_markers():
+    """
+    Make sure that our internal conversion from world to pixel
+    coordinates doesn't mess anything up.
+    """
+    fake_image = np.random.randn(2000, 2000)
```

Comment:
Do we really need a full-blown 2k by 2k image for the test? Can it be smaller, like 100 by 100?